extension/src/service/web-component-service.js

```diff
@@ -105,6 +105,22 @@ function getCarbonOffsetCosts(merchantAccount, getCarbonOffsetCostsRequestObj) {
   )
 }
 
+function updateAmount(
+  merchantAccount,
+  commercetoolsProjectKey,
+  amountUpdatesRequestObj
+) {
+  const adyenCredentials = config.getAdyenConfig(merchantAccount)
+  // extendRequestObjWithMetadata(amountUpdatesRequestObj, commercetoolsProjectKey)
+  const paymentPspReference = amountUpdatesRequestObj.paymentPspReference
+  return callAdyen(
+    `${adyenCredentials.apiBaseUrl}/payments/${paymentPspReference}/amountUpdates`,
```

Comment:
Better check the existence of `paymentPspReference` and `apiBaseUrl`? Otherwise it makes an invalid API call and simply waiting for 404 error.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/983/commits/648b2c9cd8f5824915e991365597113c9ea3e918

Reply:
No need to check `apiBaseUrl` here because this is a config parameter, not the one that comes from a user.